frame/transaction-payment/src/lib.rs

```diff
@@ -199,8 +199,8 @@ impl<T: Trait> Module<T> where
 
 			// the adjustable part of the fee
 			let adjustable_fee = len_fee.saturating_add(unadjusted_weight_fee);
-			let targeted_fee_adjustment = NextFeeMultiplier::get();
-			let adjusted_fee = targeted_fee_adjustment.saturating_mul_acc_int(adjustable_fee);
+			let targeted_fee_adjustment = Self::next_fee_multiplier();
+			let adjusted_fee = targeted_fee_adjustment.saturating_mul_int(adjustable_fee);
```

Comment:
This seems wrong. Now we have
```
adjusted_fee = targeted_fee_adjustment * adjustable_fee
```

instead of 
```
adjusted_fee = adjustable_fee + targeted_fee_adjustment * adjustable_fee
```

Reply:
In his PR notes:

> Removes the +1 (multiply_accumulate) logic when applying the multiplier. Henceforth, the multiplier will be only positive in [0, inf].

Reply:
I see. So this is intended. Is it really necessary to do this? Cause that brakes off-chain calculation and we need to support all versions of the calculation by switching on the runtime versions. I am not saying that this is not possible just that it would be preferable for off-chain calculation if this could be fixed just by changing how the `Multiplier` updates.

Reply:
My 2 cents: I totally feel for the pain of breaking changes and backwards compatibility. That being said, its my personal opinion that the (n+1) is super confusing, and knowing that this change will not only effect us, but all the future chains building on substrate, I think it is best to nip this in the bud while we are still early in the ecosystem

Reply:
I agree with what Shawn, I rather break things now. The more we wait, more chains will build upon the old system and at some point it will be beyond fixing.

Reply:
It's fine if you feel strongly about that. Just wanted to bring that up.